frontend/admin/webpack.base.js

```diff
@@ -154,8 +154,8 @@ module.exports = {
   },
   output: {
     publicPath: "/admin/", // final path for routing
-    filename: "[name].js?id=[contenthash]", // file hashing for cache busting
-    chunkFilename: "[name].js?id=[contenthash]",
+    filename: "[contenthash].[name].js", // file hashing for cache busting
```

Comment:
Might be easier to find files in alphabetical listing if it was [name].[contenthash].js instead, no?  Is there an advantage to starting with the hash?

Reply:
I can't think of any advantage, I was just using the referenced example. But, I do agree moving it to the end makes it more human friendly :)

Updated in [move content hash to end of filename](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4988/commits/06e5fcfb5330180c39eff076e76c647df73dd515)